### ISSUE TITLE
Adding to hash table example doesn't work

### ISSUE THREAD

**Issue by KosherSalt** [PowerShell/PowerShell-Docs#1943]
Issue Details
-----------------
The example code on how to add a hash table doesn't work correctly. 
```$hash = $hash.Add("Time", "Now")```

To show this in action, we can run the below.
```
$hash = @{}
$hash.GetType()
$hash = $hash.Add("Time", "Now")
$hash.GetType()
$hash -eq $null
```
Results of running above
```
PS C:\> $hash = @{}
PS C:\> $hash.GetType()

IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     True     Hashtable                                System.Object


PS C:\> $hash = @{}
PS C:\> $hash = $hash.Add("Time", "Now")

# Running gettype again after doing the add reveals that the variable is empty
PS C:\> $hash.GetType()
You cannot call a method on a null-valued expression.
At line:2 char:1
+ $hash.GetType()
+ ~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [], RuntimeException
    + FullyQualifiedErrorId : InvokeMethodOnNull

PS C:\> $hash -eq $null
True
```

Version(s) of document impacted
------------------------------
- [x] Impacts 6 document
- [x] Impacts 5.1 document
- [x] Impacts 5.0 document
- [x] Impacts 4.0 document
- [x] Impacts 3.0 document

Reason(s) for not selecting all version of documents
----------------------------------------------------
- [ ] The documented feature was introduced in selected version of PowerShell
- [ ] This issue only shows up in selected version of the document

**Comment by KosherSalt** [comment]
I've tested this in PowerShell versions 2, 3, 5.1, and 6. I'm assuming 4 and 5 have this as well

**Comment by sdwheeler** [comment]
The example in the article was incorrect. The example is fixed in PR #1944.
